lib/drivers/native/config_windows.go

```diff
@@ -0,0 +1,218 @@
+//go:build windows
+
+/**
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+package native
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/shirou/gopsutil/v3/cpu"
+	"github.com/shirou/gopsutil/v3/mem"
+
+	"github.com/adobe/aquarium-fish/lib/log"
+)
+
+type Config struct {
+	//TODO: Users []string `json:"users"` // List of precreated OS user names in format "user[:password]" to run the workload
+
+	TarPath     string `json:"tar_path"`     // Path to the tar (unpacking images) binary
+	MountPath   string `json:"mount_path"`   // Path to the mount (list of mounted volumes) binary
+	KillallPath string `json:"killall_path"` // Path to the killall (send signals to multiple processes) binary
+	RmPath      string `json:"rm_path"`      // Path to the rm (cleanup after execution) binary
+
+	ImagesPath    string `json:"images_path"`    // Where to store/look the environment images
+	WorkspacePath string `json:"workspace_path"` // Where to place the env disks
+
```

Comment:
Probably the below lines in the struct we will need to unify by using general config.go with something like: https://go.dev/play/p/6Is8ydShqSp
* config_windows.go:
```
type Config struct {
    ConfigBase
    // code specific for platform
}
```
And then in config.go:
```
type ConfigBase struct {
    // Common stuff for the configs to prevent too much of duplication
}
```

Reply:
Or you can try vice-versa, but you will need to run special functions for validation on platform-specific logic like: c.ConfigPlatform.Validate()

Reply:
https://eli.thegreenplace.net/2020/embedding-in-go-part-1-structs-in-structs - here is better then I can describe.